server/controllers/store.js

```diff
@@ -1,25 +1,23 @@
-import express from 'express';
-
+/* eslint-disable class-methods-use-this */
+import express from 'express'
 import { product } from '../helpers/productarray';
 import { Order } from '../helpers/productarray';
-import { catigory } from '../helpers/productarray';
+import { category } from '../helpers/productarray';
 import { Prods } from '../models/productsModel';
-import { category } from '../models/productsModel';
+import { categoryarray } from '../models/productsModel';
 import { orders } from '../models/productsModel';
-import queries from '../models/queries';
-
-export default class Products {
 
-  static async addProduct(req, res) {
-    const newProduct = queries.addProduct;
-    const products = await (newProduct);
-    return res.status(201).json({
+class Products {
+  addProduct(req, res) {
+    const {name, category, price, size } = req.body;
+    const newProduct = product.push(new Prods(product.length+1, name, category, price, size));
       success: 'true',
       message: 'product added successfuly',
       product,
     });
   }
 
+
   static async allProducts(req, res) {
```

Comment:
I believe you should name this getAllProducts

Reply:
renamed